src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/credit-card-number/page.tsx

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { HighRiskBreachLayout } from "../HighRiskBreachLayout";
+
+export default function CreditCardDataBreach() {
```

Comment:
Below we were checking for sessions, do we need to check that the user is logged in for these files?

```js
  const session = await getServerSession(authOptions);
  if (!session?.user?.subscriber?.id) {
    return redirect("/");
  }
```